### DISCUSSION TITLE
Hiding the scroll bar

### DISCUSSION THREAD

**Discussion by aagha** [aristocratos/btop#571] [General]
When going full-screen on btop, I'd like to hide the scroll bar:

![image](https://github.com/aristocratos/btop/assets/5522103/a4c59094-9091-474c-82ea-b1fc8123c6a4)

I suspect this might be a function of the Windows terminal setting(?), but can't figure out which one:

![image](https://github.com/aristocratos/btop/assets/5522103/9d238cd0-ea5d-4b9a-bd5c-7bfc08e1de4c)

...or, is there another way of doing this?

**Comment by hrxn** [comment]
Yeah, it's the terminal..

Dunno, you could try to use the new Windows Terminal, the scrollbar can be always hidden there with a setting..
